src/components/order-service/order-service.controller.ts

```diff
@@ -12,10 +12,12 @@ export class OrderServiceController implements angular.IController {
   }
 
   public $onInit() {
-    this.ctrl.serviceIcon = this.ctrl.service.imageURL ? this.ctrl.service.imageURL : 'pf pficon-service';
-    this.ctrl.serviceName = this.ctrl.service.displayName ? this.ctrl.service.displayName : this.ctrl.service.metadata.name;
-    this.ctrl.versions = this.ctrl.service.versions;
-    this.ctrl.selectedVersion = this.ctrl.service.versions[0];
+    this.ctrl.serviceIcon = this.ctrl.service.icon ? this.ctrl.service.icon : 'pf pficon-service';
```

Comment:
osbMetadata, its a raw extension blob which means that data is just getting passed back from the underlying info the broker sent to the service catalog API. You wont see the fields called out in the svc cat schema because of that.  There are no guarantees that any of that metadata will be there, its all just done by convention.